Tools/Scripts/run-jsc-stress-tests

```diff
@@ -2538,7 +2552,8 @@ def handleCollectionFile(collection)
                 | pathToSearch |
                 allJSFiles(pathToSearch).each {
                     | path |
-                    
+
+                    next if not $isWasmPlatform and isWasmPath(path)
```

Comment:
`isWasmPath` is not defined so we should use `isWasmTest` ?

Reply:
Thanks.